gcloud-java-pubsub/src/main/java/com/google/gcloud/pubsub/TopicInfo.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.pubsub;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.base.MoreObjects;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Pub/Sub Topic information.
+ */
+public class TopicInfo implements Serializable {
```

Comment:
This is obviously consistent with other client libraries gcloud-java has, but I do wonder if we could create a topic with a string and only have the Topic class (unless we plan to have more topic metadata).  I don't feel particularly strongly, I just figured I'd raise the idea.